python/cudf/cudf/core/reshape.py

```diff
@@ -964,6 +969,21 @@ def unstack(df, level, fill_value=None):
     a
     1     5  <NA>     6  <NA>     7
     2  <NA>     8  <NA>     9  <NA>
+
+    Unstacking single level index dataframe:
+
+    >>> df.unstack(['b', 'd']).unstack()
```

Comment:
I think this example is a little opaque - it's sometimes difficult to visualize exactly what the result of `unstack` should be for even a single level, and here I find it a little hard to connect to dots through the chained operation. I'd recommend an example that starts with a dataframe with a single index and shows the result of unstacking that dataframe into a series instead.